app/scripts/lib/safe-reload.ts

```diff
@@ -12,11 +12,20 @@ const { sentry } = global;
  * @param persistenceManager - The PersistenceManager instance to be used for
  * updates.
  */
```

Comment:
Nit: Could use documentation for the type param:

```suggestion
 * @template Type - The persistence manager type.
 */
```